examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -146,6 +146,63 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 		}
 	});
 
+	interface UnregisterWebhookRequest extends Request {
+		body: {
+			// The target URL to unsubscribe from change notifications
+			url: string;
+		};
+	}
+	/**
+	 * Unregister's the sender's URL from receiving notifications when the external task-list data changes.
+	 *
+	 * Expected input data format: {@link UnregisterWebhookRequest}
+	 */
+	expressApp.post("/unregister-webhook", (request: UnregisterWebhookRequest, result) => {
```

Comment:
I like this pattern of adding explicit types! Good call! We may want to migrate our other hooks to do this in the future.